istioctl/pkg/auth/listener.go

```diff
@@ -86,17 +90,30 @@ func getHTTPConnectionManager(filter *listener.Filter) *hcm_filter.HttpConnectio
 func getHTTPFilterConfig(filter *hcm_filter.HttpFilter, out proto.Message) error {
 	switch c := filter.ConfigType.(type) {
 	case *hcm_filter.HttpFilter_Config:
-		if err := util.StructToMessage(c.Config, out); err != nil {
+		if err := conversion.StructToMessage(c.Config, out); err != nil {
 			return err
 		}
 	case *hcm_filter.HttpFilter_TypedConfig:
-		if err := types.UnmarshalAny(c.TypedConfig, out); err != nil {
+		if err := ptypes.UnmarshalAny(c.TypedConfig, out); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+func StructToGoGoMessage(pbst *structpb.Struct, out proto.Message) error {
```

Comment:
unused function

Reply:
do you know why linter is not catching this? it used to be able to detect unused functions.

Reply:
Probably public functions don't get the check